File: samples/snippets/visualbasic/VS_Snippets_CLR_System/system.reflection.assemblyname.getassemblyname/vb/getassemblyname1.vb
' Visual Basic .NET Document
Option Strict On

' <Snippet1>
Imports System.Reflection

Public Module Example
   Public Sub Main
      Console.WriteLine(AssemblyName.GetAssemblyName(".\UtilityLibrary.dll"))
   End Sub
End Module
' The example displays output like the following:
'   UtilityLibrary, Version=1.1.0.0, Culture=neutral, PublicKeyToken=null
' </Snippet1>

File: samples/snippets/csharp/VS_Snippets_CLR_System/system.reflection.assemblyname.getassemblyname/cs/getassemblyname1.cs
// <Snippet1>
using System;
using System.Reflection;

public class Example
{
   public static void Main()
   {
      Console.WriteLine(AssemblyName.GetAssemblyName(@".\UtilityLibrary.dll"));
   }
}
// The example displays output like the following:
//   UtilityLibrary, Version=1.1.0.0, Culture=neutral, PublicKeyToken=null
// </Snippet1>
